src/definitions/modules/search.js

```diff
@@ -1591,8 +1603,17 @@
                     // each result
                     $.each(response[fields.results], function (index, result) {
                         html += result[fields.url]
-                            ? '<a class="result" href="' + result[fields.url].replace(/"/g, '') + '">'
-                            : '<a class="result">';
+                            ? '<a href="' + result[fields.url].replace(/"/g, '') + '" '
+                            : '<div ';
+
+                        html += result[fields.id] !== undefined
+                            ? ' id="' + result[fields.id] + '" '
```

Comment:
Needs to be sanitized (double quotes removed)
```suggestion
                            ? ' id="' + result[fields.id].replace(/"/g, '') + '" '
```